src/NHSD.BuyingCatalogue.Organisations.Api/Startup.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using NHSD.BuyingCatalogue.Organisations.Api.Data;
+using NHSD.BuyingCatalogue.Organisations.Api.Repositories;
+using Serilog;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddTransient<IOrganisationRepository, OrganisationRepository>();
+
+            services.AddDbContext<ApplicationDbContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("CatalogueUsers")));
+
+            var authority = Configuration.GetValue<string>("authority");
+
+            foreach (IConfigurationSection configurationSection in Configuration.GetChildren())
+            {
+                Log.Logger.Information("{Key} = {Value}", configurationSection.Key, configurationSection.Value);
+            }
+
+            services.AddAuthentication("Bearer")
+                .AddJwtBearer("Bearer", options =>
+                {
+                    options.Authority = authority;
+                    options.RequireHttpsMetadata = false;
+                    options.Audience = "SampleResource";
```

Comment:
Should be "Organisation". Please make sure this extra scope is requested  used everywhere